clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/ops/servergroup/DeployOpenstackAtomicOperation.groovy

```diff
@@ -110,8 +110,8 @@ class DeployOpenstackAtomicOperation implements AtomicOperation<DeploymentResult
 
       task.updateStatus BASE_PHASE, "Successfully created server group."
 
-      deploymentResult.deployedNames = Arrays.asList(stackName)
-      deploymentResult.deployedNamesByLocation = [(description.region): deploymentResult.deployedNames]
+      deploymentResult.serverGroupNames = Arrays.asList("$description.region:$stackName".toString()) //stupid GString
```

Comment:
Ha, silly gStrings.

Any reason why you used `Arrays.asList` instead of just wrapping the string in `[]`?